operations/mimir-mixin/dashboards/reads.libsonnet

```diff
@@ -78,12 +78,23 @@ local filename = 'mimir-reads.json';
         ),
       )
       .addPanel(
-        $.panel('Label queries / sec') +
-        $.statPanel($.queries.query_frontend.labelQueriesPerSecond, format='reqps') +
+        $.panel('Label names queries / sec') +
```

Comment:
I'm not sure about this:
"label names queries" sounds strange to me because of the double plural, shouldn't it rather be "label name queries"?

Reply:
But they're "queries that return label names", not "queries that return label name".

Reply:
yes, i know, that's why I wasn't sure. 
I think you should discuss that with a native english speaker, not with me :)

Reply:
Let's ask @osg-grafana

Reply:
Label names’ queries

Reply:
Alternatively "Label names from queries / sec"

Reply:
This is an odd one.

Reply:
In a full sentence or two, what does "Labels names queries / sec" mean?
* "The number of label names that the query endpoint returns per second?"
* "No. of label names from query endpoint / sec"?
* "No. of label names per query endpoint / sec"?
* "Labels-names-query / sec"? (and expand the definition somehow in the UI)?

Honestly this one is stumping me and is nearly guaranteed to confuse the user.

Reply:
How about we just completely drop the `queries` part from the legend? The panel title already states that this is about queries.

Reply:
> In a full sentence or two, what does "Labels names queries / sec" mean?

One of the endpoints of the service is "Label names", this is the legend for the amount of queries to that endpoint.
So it's the last option, I guess:

> "Labels-names-query / sec"?

(But singular _label_, not _labels_)

---

How about we just completely drop the queries part from the legend? The panel title already states that this is about queries.

I agree that it's repetitive, but it would be even more misleading if we had legend just for "label names", "label values"...

Reply:
I have quoted the `"Label names" queries` and `"Label values" queries` in the descriptions and legends.

Reply:
let's add another cook 😄

since they are endpoints we can have the endpoint suffix in the legend. `labels`, `label_name_values`, `query`, `query_range`. And the panel can be called "Request rate by endpoint"

I don't think it's more confusing than the plain English versions - these are the external APIs that Mimir exposes (almost) verbatim.

Reply:
I think we're blocked here. I will merge this as-is, to unblock the feature, then we can spend time on the best wording for this. It's not worse than the current version that just puts everything in the same bucket.